resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -439,18 +439,28 @@ export default {
             Events.$emit('images.change', this.imageId, this.image);
         },
         cachePreviousAndNext() {
+            // currently const number of images to cache
+            const imagesToLoad = 10;
+            // Array with all promises
+            let idsToCache = [];
+
             let previousId = this.imagesIds[this.getPreviousIndex(this.imageIndex)];
-            let nextId = this.imagesIds[this.getNextIndex(this.imageIndex)];
+
+            // load next X ids and push Annotation and Image in array for promise all
+            for (const x of Array(imagesToLoad).keys()) {
+                const currentId = this.imagesIds[this.getNextIndex(this.imageIndex + x)]
```

Comment:
Does this also handle the case where `this.imageIndex + x` is larger than the actual number of images?

Reply:
> So this PR implements both the "Shift+Scroll" and the "load x next images" features?

Unfortunately yes. This was a git mistake on my part. 

> Does this also handle the case where this.imageIndex + x is larger than the actual number of images?

Yes it works fine for one image or less than should be loaded (currently 10). 
I currently change the code, so if the amount of images to load is greater than the amount of images, it only loads as many as there are.